packages/reactstrap-validation-select/AvSelect.js

```diff
@@ -16,9 +16,9 @@ const {
   ClearIndicator,
 } = reactSelectComponents;
 
-const createOption = (label, labelkey = 'label') => ({
-  [labelkey]: label,
-  value: label.toLowerCase().replace(/\W/g, ''),
+const createOption = (label, labelKey = 'label', valueKey = 'value') => ({
+  [labelKey]: label,
+  [valueKey]: label.toLowerCase().replace(/\W/g, ''),
```

Comment:
Can we add a quick test that has `creatable` and a custom `valueKey` to make sure this will work properly for future updates.